cypress/Dockerfile

```diff
@@ -33,20 +33,15 @@ RUN yum install -y unzip
 
 RUN npm install yarn -g
 
-RUN mkdir -p ~/.cache/Cypress/8.3.0/Cypress && \
-    wget --no-check-certificate http://download.cypress.io/desktop/8.3.0?platform=linux -O /tmp/z.$$ && \
-    unzip -d ~/.cache/Cypress/8.3.0/ /tmp/z.$$ && \
-    rm /tmp/z.$$ && \
-    chmod +x ~/.cache/Cypress/8.3.0/Cypress/Cypress && \
-    rm -rf /var/lib/apt/lists/*
-
 COPY / hmda-frontend/
 
-WORKDIR hmda-frontend/
+WORKDIR /hmda-frontend/
 
 RUN yarn
 
 RUN useradd -ms /bin/bash hmda_cypress_user && \
   chown -R hmda_cypress_user:hmda_cypress_user /hmda-frontend/
 
 USER hmda_cypress_user
+
+RUN yarn cypress install
```

Comment:
nice, I'm guessing this replaces the need to do all the downloading explicitly done by the block you removed above?

Reply:
Yes. The new command handles the download/installation to appropriate directory for the `hmda_cypress_user`. With the previous install method, the Cypress binary seemed to be getting overwritten, which might just have been an order-of-operations issue, but I think having this new command provides a much more succinct, understandable, and maintainable approach.